db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
```

Comment:
sorry, don't ignore this. it says - it is strongly recommended that you check this file into your version control system. So that on another system we can load the schema instead of running all the migrations. More info on the comments in this file.